pkg/social/common.go

```diff
@@ -23,24 +23,26 @@ func isEmailAllowed(email string, allowedDomains []string) bool {
 	return valid
 }
 
-func HttpGet(client *http.Client, url string) ([]byte, error) {
+func HttpGet(client *http.Client, url string) ([]byte, http.Header, error) {
```

Comment:
Is there a reason not to use named return values other than backwards compatibility to pre-1.7? Would remove the need to specify nil and err in all return statements.